cmd/cosign/cli/verify/verify_attestation.go

```diff
@@ -121,6 +123,24 @@ func (c *VerifyAttestationCommand) Exec(ctx context.Context, images []string) (e
 		if err != nil {
 			return errors.Wrap(err, "loading certificate from reference")
 		}
+		// Verify certificate with chain
+		// First intermediate at chain[0], root at chain[n-1]
```

Comment:
should we factor this logic out into something shared? I think we probably have this in a few other places too...

Reply:
Good suggestion! I added a function that takes the chain and calls `ValidateAndUnpackCert`. This has the added benefit of also verifying the OIDC subject/issuer flags too.

Reply:
Nice!